core/systems/graphite/_public.ex

```diff
@@ -79,6 +79,14 @@ defmodule Systems.Graphite.Public do
     |> Repo.transaction()
   end
 
+  def update_leaderboard(leaderboard, attrs) do
+    Multi.new()
+    |> Multi.update(:graphite_leaderboard, fn _ ->
+      Graphite.LeaderboardModel.changeset(leaderboard, attrs)
+    end)
```

Comment:
Consider to add a signal dispatch and possibly update some liveviews in reaction to this signal.